src/whitenoise/accounts/mod.rs

```diff
@@ -1049,6 +1081,60 @@ impl Whitenoise {
 
         Ok(())
     }
+
+    /// Helper method to get relay URLs from database for testing purposes
+    #[cfg(test)]
+    pub async fn get_account_relays_db(
+        &self,
+        pubkey: &PublicKey,
+        relay_type: RelayType,
+    ) -> Result<DashSet<RelayUrl>> {
+        let account = self.load_account(pubkey).await?;
+
+        let relays = match relay_type {
+            RelayType::Nostr => account.nip65_relays,
+            RelayType::Inbox => account.inbox_relays,
+            RelayType::KeyPackage => account.key_package_relays,
+        };
+
+        Ok(relays)
+    }
```

Comment:
Not a fan of these helper methods for tests. This one looks like it would be useful though in general, instead of using the `account.<relay_type>`, we can probably use the more general method and avoid code duplication (e.g.: iterating on relay types)